docs/data-sources/infoblox_zone_forward.md

```diff
@@ -0,0 +1,85 @@
+# Zone Forward Data Source
+
+Use the `infoblox_zone_forward` data source to retrieve the following information for Forward Zone if any, which are managed by a NIOS server:
+
+* `fqdn`: The name of this DNS zone. For a reverse zone, this is in “address/cidr” format. Example: `11.10.0.0/24`. For other zones, this is in FQDN format. Example: `demozone.com` This value can be in unicode format.
+* `view`: The name of the DNS view in which the zone resides. Example: `external`.
+* `zone_format`: Determines the format of corresponding zone. Valid values are `FORWARD`, `IPV4` and `IPV6`.
+* `comment`: The Description of Forward Zone Object. Example: `random forward zone`.
+* `ext_attrs`: the set of extensible attributes of the record, if any. The content is formatted as string of JSON map. Example: `"{\"Location\":\"unknown\",\"TestEA\":\"ZoneTesting\"}"`.
+
+For usage of filters, add the fields as keys and appropriate values to be passed to the keys like `fqdn`, `view` corresponding to object.
+From the below list of supported arguments for filters,  use only the searchable fields for retrieving the matching records.
+
+### Supported Arguments for filters
+
+-----
+| Field       | Alias       | Type   | Searchable |
+|-------------|-------------|--------|------------|
+| fqdn        | fqdn        | string | yes        |
+| view        | view        | string | yes        |
+| zone_format | zone_format | string | yes        |
+| comment     | comment     | string | yes        |
+
+
+!> Any of the combination from searchable fields in supported arguments list for fields are allowed.
+
+!> Please consider using only fields as the keys in terraform datasource filters, kindly don't use alias names as keys from the above table.
```

Comment:
Lets rephrase as per official doc - "Aliases are the parameter names used in the prior releases of Infoblox IPAM Plug-In for Terraform. Do not use the alias names for parameters in the data source blocks. Using them can result in error scenarios."